server/shell.nix

```diff
@@ -0,0 +1,45 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+let
+
+  podmanSetupScript = let
+    registriesConf = pkgs.writeText "registries.conf" ''
+      [registries.search]
+      registries = ['docker.io']
+      [registries.block]
+      registries = []
+    '';
+  in pkgs.writeScript "podman-setup" ''
+    #!${pkgs.runtimeShell}
+    if ! test -f ~/.config/containers/policy.json; then
+      install -Dm555 ${pkgs.skopeo.src}/default-policy.json ~/.config/containers/policy.json
+    fi
+    if ! test -f ~/.config/containers/registries.conf; then
+      install -Dm555 ${registriesConf} ~/.config/containers/registries.conf
+    fi
+  '';
+
+in pkgs.mkShell {
+
+  buildInputs = [
+    pkgs.cargo-make
+    pkgs.clippy
+    pkgs.openssl
+    pkgs.pkg-config
+    pkgs.podman
+    pkgs.postgresql
+    pkgs.skopeo
+
+    # Not on Darwin:
```

Comment:
you can use `++ lib.optionals !stdenv.isDarwin [ #packages not to use on mac#]` to automate this!

Reply:
@jenr24G -- nice suggestion. Though does this depend on whether we want a static or dynamic approach to package exclusion?

Reply:
I'll put this in the next version, Jen! Thanks for the tip!